css/style.css

```diff
@@ -934,12 +934,13 @@
 #app-sidebar div.content-wrapper .body .section input[type="range"] {
   border: medium none;
   box-shadow: none;
-  width: 190px;
-  margin: 5px 36px 0 0;
+  width: calc(100% - 90px);
+  margin: 0 36px 0 4px;
```

Comment:
Looks like this for me with 5px 36px 0 4px then:
![localhost-index php-apps-tasks- nexus 5x](https://cloud.githubusercontent.com/assets/1389813/25896365/249ecc00-3584-11e7-9684-3a525bcb761f.png)